client/my-sites/patterns/pages/category/index.tsx

```diff
@@ -47,22 +47,24 @@ export const PatternsCategoryPage = ( {
 			/>
 
 			{ categoryNavList && (
-				<CategoryPillNavigation
-					selectedCategory={ category }
-					buttons={ [
-						{
-							icon: ImgStar,
-							label: 'Discover',
-							link: addLocaleToPathLocaleInFront( '/patterns' ),
-						},
-						{
-							icon: ImgGrid,
-							label: 'All Categories',
-							link: '/222',
-						},
-					] }
-					list={ categoryNavList }
-				/>
+				<div className="category-pill-navigation__patterns-wrapper">
```

Comment:
```suggestion
				<div className="patterns-page-category__pill-navigation">
```
I think it's slightly misleading to use the `category-pill-navigation` "BEM namespace" outside of `CategoryPillNavigation`. Granted, this element isn't a descendant of `.patterns-page-category`, but I still think something like this would be a more appropriate class name here.

Reply:
Yeah, awesome point, totally agree 👍